### PR TITLE
Add support for Rails 6.0.0.rc1

### PR DESCRIPTION
**Pull request by jtapia**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,11 +10,14 @@ env:
 rvm:
   - 2.6.2
   - 2.3.1
+addons:
+  postgresql: 9.4
```
Comment (parndt) [inline .travis.yml]: Can you say why you added this @jtapia ?

**Comment by parndt** [comment]
Thanks @jtapia

File: .travis.yml (hunk 1 above)
Comment (jtapia) [reply .travis.yml]: this a fix for this issue https://travis-ci.org/collectiveidea/awesome_nested_set/jobs/540578878,  Rails 6 drops support for PostgreSQL version less than 9.3 https://github.com/rails/rails/pull/34520

File: gemfiles/rails_6.0.gemfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# This file was generated by Appraisal
+
+source "https://rubygems.org"
+
+gem "rails", "~> 6.0.0rc1"
```
Comment (parndt) [inline gemfiles/rails_6.0.gemfile]:
```suggestion
gem "rails", "~> 6.0.0.rc1"
```

File: Appraisals [hunk 3]
```diff
@@ -1,4 +1,4 @@
-{ '4.0' => '4.0.0', '4.1' => '4.1.0', '4.2' => '4.2.0', '5.0' => '5.0.0', '5.1' => '5.1.0' }.each do |rails, version|
+{ '4.0' => '4.0.0', '4.1' => '4.1.0', '4.2' => '4.2.0', '5.0' => '5.0.0', '5.1' => '5.1.0', '6.0' => '6.0.0rc1' }.each do |rails, version|
```
Comment (parndt) [inline Appraisals]:
```suggestion
{ '4.0' => '4.0.0', '4.1' => '4.1.0', '4.2' => '4.2.0', '5.0' => '5.0.0', '5.1' => '5.1.0', '6.0' => '6.0.0.rc1' }.each do |rails, version|
```

Comment (jtapia) [reply Appraisals]: done https://github.com/collectiveidea/awesome_nested_set/pull/404/files#diff-3bb7346aca79d8142e16e61372bd04dcR1

File: gemfiles/rails_6.0.gemfile (hunk 2 above)
Comment (jtapia) [reply gemfiles/rails_6.0.gemfile]: done https://github.com/collectiveidea/awesome_nested_set/pull/404/files#diff-f6a6f2ef81cbd15493b9768403332f9aR5

**Comment by parndt** [comment]
Thank you @jtapia I've merged that in :tada:

**Comment by damianlegawiec** [comment]
Hey @parndt any plans on releasing a new version containing this Rails 6 fixes? Thanks

**Comment by parndt** [comment]
@damianlegawiec Thanks! I'm not keen on releasing a new version to support Rails 6, until Rails 6 is actually released. For now you can use:

```ruby
gem 'awesome_nested_set', git: 'https://github.com/collectiveidea/awesome_nested_set', branch: 'master'
```

**Pull request merged**